lib/matplotlib/pyplot.py

```diff
@@ -896,72 +931,128 @@ def gca(**kwargs):
+    polar : boolean, optional
+        If True, equivalent to projection='polar'.
 
-    If *nrows*, *ncols* and *index* are all less than 10, they can also be
-    given as a single, concatenated, three-digit number.
+    sharex, sharey : `~.axes.Axes`, optional
+        Make the returned axes share the x or y `~matplotlib.axis`
+        with the input axes.
 
-    For example, ``subplot(2, 3, 3)`` and ``subplot(233)`` both create an
-    `matplotlib.axes.Axes` at the top right corner of the current figure,
-    occupying half of the figure height and a third of the figure width.
+    label : str
+        A label for the returned axes.
 
-    .. note::
+    Other Parameters
+    ----------------
+    **kwargs
+        This method also takes the keyword arguments for
+        the returned axes base class. The keyword arguments for the
+        rectilinear base class `~.axes.Axes` can be found in
+        the following table but there might also be other keyword
+        arguments if another projection is used.
+        %(Axes)s
 
-       Creating a subplot will delete any pre-existing subplot that overlaps
-       with it beyond sharing a boundary::
+    Returns
+    -------
+    axes : an `.axes.SubplotBase` subclass of `~.axes.Axes` (or a subclass \
+    of `~.axes.Axes`)
 
-          import matplotlib.pyplot as plt
-          # plot a line, implicitly creating a subplot(111)
-          plt.plot([1,2,3])
-          # now create a subplot which represents the top plot of a grid
-          # with 2 rows and 1 column. Since this subplot will overlap the
-          # first, the plot (and its axes) previously created, will be removed
-          plt.subplot(211)
-          plt.plot(range(12))
-          plt.subplot(212, facecolor='y') # creates 2nd subplot with yellow background
+        The axes of the subplot. The returned axes base class depends on
+        the projection used. It is `~.axes.Axes` if rectilinear projection
+        are used and `.projections.polar.PolarAxes` if polar projection
+        are used. The returned axes is then a subplot subclass of the
+        base class.
 
-       If you do not want this behavior, use the
-       :meth:`~matplotlib.figure.Figure.add_subplot` method or the
-       :func:`~matplotlib.pyplot.axes` function instead.
+    Notes
+    -----
+    Creating a subplot will delete any pre-existing subplot that overlaps
+    with it beyond sharing a boundary::
+
+        import matplotlib.pyplot as plt
+        # plot a line, implicitly creating a subplot(111)
+        plt.plot([1,2,3])
+        # now create a subplot which represents the top plot of a grid
+        # with 2 rows and 1 column. Since this subplot will overlap the
+        # first, the plot (and its axes) previously created, will be removed
+        plt.subplot(211)
+
+    If you do not want this behavior, use the `.Figure.add_subplot` method
+    or the `.pyplot.axes` function instead.
```

Comment:
?? You just said these also have that behaviour....

Reply:
The confusing bit is that `plt.subplot` will kill any other subplot that's below it, **unless**  that subplot has been created with identical `*args,**kwargs`. 

Say you have a subplot

    ax1 = plt.subplot(111)

Then 

    ax2 = plt.subplot(111, label="label")

will kill `ax1`, while

    ax2 = plt.subplot(111) 

will activate `ax1` (hence `ax1 == ax2`)